packages/common/docs/classes/default.md

```diff
@@ -1579,6 +1579,8 @@ Note that these supported custom chains only provide some base parameters (usual
 network ID and a name) and can only be used for selected use cases (e.g. sending a tx with
 the `@ethereumjs/tx` library to a Layer-2 chain).
 
+Note: For Optimistic Kovan and Optimistic Ethereum, the London hardfork has not been implemented yet so attempts to send EIP1559 transactions will fail.
```

Comment:
you can't add this directly to `docs` because it will be overwritten on `docs:build`, so add it to the source typedoc instead. we typically rebuild the docs prior to release but you can do it in this PR as well.

can you also add to this table in the readme? https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/tx#l2-support

Reply:
Done, fixed